core/domain/html_validation_service.py

```diff
@@ -238,7 +238,7 @@ def convert_to_textangular(html_data):
         elif tag.name == 'td' and tag.next_sibling:
             tag.insert_after(' ')
             tag.unwrap()
-        # div and table rows both are replaced with p tag
+        # Rows div and table both are replaced with p tag
```

Comment:
Perhaps: "Divs and table rows are both replaced with ..."

(I don't think "div rows" is a thing.)

Reply:
Done